301-nested-vms-in-virtual-network/README.md

```diff
@@ -0,0 +1,101 @@
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F301-nested-vms-in-virtual-network%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png"/>
+</a>
+
+## Final Configuration
+
+The environment in this guide has the below configurations. This section is intended to be used as a reference.
+
+1. Azure Virtual Network Information.
+    + VNet High Level Configuration.
+        + Name: Nested-Fun
+        + Address Space: 10.0.0.0/22
+        + Note: This will be made up of four Subnets. Also, these ranges are not set in stone. Feel free to address your environment however you want.
+
+    + First Subnet High Level Configuration.
+        + Name: NAT
+        + Address Space: 10.0.0.0/24
+        + Note: This is where our Hyper-V hosts primary NIC resides. This will be used to handle outbound NAT for the nested VMs. It will be the gateway to the internet for your nested VMs.
+
+    + Second Subnet High Level Configuration.
+        + Name: Hyper-V-LAN
+        + Address Space: 10.0.1.0/24
+        + Note:  Our Hyper-V host will have a second NIC that will be used to handle the routing between the nested VMs and non-internet resources external to the Hyper-V host.
+
+    + Third Subnet High Level Configuration.
+        + Name: Ghosted
+        + Address Space: 10.0.2.0/24
+        + Note:  This will be a “floating” subnet. The address space will be consumed by our nested VMs and exists to handle route advertisements back to on-premises. No VMs will actually be deployed into this subnet.
+
+    + Fourth Subnet High Level Configuration.
+        + Name: Azure-VMs
+        + Address Space: 10.0.3.0/24
+        + Note: Subnet containing Azure VMs.
+
+2. Our Hyper-V host has the below NIC configurations.
+    + Primary NIC
+        + IP Address: 10.0.0.4
+        + Subnet Mask: 255.255.255.0
+        + Default Gateway: 10.0.0.1
+        + DNS: Configured for DHCP
+        + IP Forwarding Enabled: No
+
+    + Secondary NIC
+        + IP Address: 10.0.1.4
+        + Subnet Mask: 255.255.255.0
+        + Default Gateway: Empty
+        + DNS: Configured for DHCP
+        + IP Forwarding Enabled: Yes
+
+    + Hyper-V Created NIC for Internal Virtual Switch
+        + IP Address: 10.0.2.1
+        + Subnet Mask: 255.255.255.0
+        + Default Gateway: Empty
+
+3. Our Route Table will have a single rule.
+    + Rule 1
+        + Name: Nested-VMs
+        + Destination: 10.0.2.0/24
+        + Next Hop: Virtual Appliance - 10.0.1.4
+
+## Post Deployment Steps
+
+Once the deployment is complete to access your Hyper-V Host an inbound security rule will need to be created on your NAT Subnet NSG.
```

Comment:
Any reason we can't do this in the template?  If not, add some more specific instructions...